src/integration-helpers.ts

```diff
@@ -1,5 +1,6 @@
 import ts from "typescript";
 import { Method } from "./method";
+import { addJsDocComment } from "./zts-helpers";
```

Comment:
this is fine, because the utility is involved into low level ZTS production of commented props